examples/mobile/build.rs

```diff
@@ -0,0 +1,27 @@
+// a build script to copy libc++_shared.so to the target directory.

```

Comment:
this shouldn't be needed, oboe handles it for us if feature `android_shared_stdcxx` is enabled, see https://github.com/katyo/oboe-rs/blob/master/sys/build.rs

Is it needed for something else?

Reply:
It's needed for the android apk(ndk will not export it, which is said in it's [readme](https://crates.io/crates/cargo-ndk))

what if I add it manually in example readme?